cogs/halloween_2022.py

```diff
@@ -166,6 +166,8 @@ async def offer(self, ctx, pokemon: commands.Greedy[converters.PokemonConverter]
     @flags.add_flag("--nickname", nargs="*", action="append")
     @flags.add_flag("--type", "--t", type=str, action="append")
     @flags.add_flag("--region", "--r", type=str, action="append")
+    @flags.add_flag("--move", nargs="+", action="append")
+    @flags.add_flag("--learns", nargs="*", action="append")
```

Comment:
Are these necessary since this event is not live anymore? (could just be me though)

Reply:
I want to keep backwards compatibility incase this code is ever reused. There's no harm in it so why not :)